### PR TITLE
Add support for handling mouse up and down on days

### PR DESCRIPTION
**Pull request by eldritchideen**
This PR adds the ability to add handlers for mouseUp and mouseDown events on days. This allows one to select dates by moving the mouse over the days while holding down the mouse button.

### CODE REVIEW

File: src/Month.js [hunk 1]
```diff
@@ -54,6 +54,8 @@ export default class Month extends Component {
     onDayKeyDown: PropTypes.func,
     onDayMouseEnter: PropTypes.func,
     onDayMouseLeave: PropTypes.func,
+    onDayMouseDown: PropTypes.func,
+    onDAyMouseUp: PropTypes.func,
```
Comment (eldritchideen) [inline src/Month.js]: Let me fix these items up.

**Comment by eldritchideen** [comment]
Added tests and eslint runs clean.

**Comment by gpbl** [comment]
Awesome! Thanks a lot 👍

**Comment by gpbl** [comment]
Published as v6.1.0

**Pull request merged**